### DISCUSSION TITLE
How to launch app from a different local Windows profile

### DISCUSSION THREAD

**Discussion by absintheboy** [LizardByte/community#575] [Sunshine]
### Select Topic Area

Question

### Body

Hi, I've been trying out DuoStream to create a headless install of Sunshine so I can still work on the PC while the kids are gaming on the TV.  I've created a new windows profile specifically to run DuoStream and while steam works great out of the box, I'm struggling to get Playnite to launch from my main profile and run on the one created for streaming.

Is this something that's even possible, launching an application installed on another windows profile?  As my steam games launch without issue, I assume it is, but I'm completely lost.

**Comment by ReenigneArcher** [comment]
We don't support commercial/paid forks of Sunshine... or any forks for that matter.
